components/Badge/badge.css

```diff
@@ -0,0 +1,69 @@
+.bdg-av{
+    display:flex;
+    flex-direction: row;
+}
+:root{
+    --online:rgb(49, 218, 49);
+    --offline:yellow;
+    --onCall:red;
+    --whislist:rgb(209, 86, 86);
+    --notification:rgb(52, 139, 221);
+}
+.badge-on-avtar{
+    position: relative;
```

Comment:
can have some padding on badges, looking too small which creates a bad UX.

Reply:
Hey you want me increase the size of badge right, but padding helps to increase space around it. However i have increased the width and height of the badge